adage/analyze/management/commands/delete_participation_type.py

```diff
@@ -0,0 +1,46 @@
+"""
+Management command that only serves to delete ParticipationType objects in
```

Comment:
Since the name is unique in the database, this command "serves to delete **a** ParticipationType object" instead of "... objects".